### PR TITLE
Japanese manual: Update tutorials and api

### PR DESCRIPTION
**Pull request by kcrt**
[ci skip]

### CODE REVIEW

File: docs-translations/jp/tutorial/online-offline-events.md [hunk 1]
```diff
@@ -9,10 +9,11 @@ const electron = require('electron');
...
 const BrowserWindow = electron.BrowserWindow;
 
-var onlineStatusWindow;
-app.on('ready', function() {
+let onlineStatusWindow;
+
+app.on('ready', () => {
   onlineStatusWindow = new BrowserWindow({ width: 0, height: 0, show: false });
-  onlineStatusWindow.loadURL('file://' + __dirname + '/online-status.html');
+  onlineStatusWindow.loadURL('file://${__dirname}/online-status.html');
```
Comment (Flatta) [inline docs-translations/jp/tutorial/online-offline-events.md]: Isn't it `file://${__dirname}/online-status.html` ?

File: docs-translations/jp/tutorial/online-offline-events.md [hunk 2]
```diff
@@ -46,13 +47,14 @@ const app = electron.app;
...
```
Comment (Flatta) [inline docs-translations/jp/tutorial/online-offline-events.md]: Isn't it `` `file://${__dirname}/online-status.html` `` ?

File: docs-translations/jp/tutorial/quick-start.md [hunk 3]
```diff
@@ -52,48 +52,60 @@ __注記__： `package.json` に `main` が存在しない場合、Electron は
...
-  });
+app.on('activate', () => {
+  // OS X では、ドックをクリックされた時にウィンドウがなければ新しく作成する。
+  if (win === null) {
+    createWindow();
+  }
 });
+
+// In this file you can include the rest of your app's specific main process
+// code. You can also put them in separate files and require them here.
```
Comment (Flatta) [inline docs-translations/jp/tutorial/quick-start.md]: 上記の英文は消し忘れではありませんか？
（もし意図的ならすみません）

**Comment by kcrt** [comment]
Thank you for your comments, @Flatta. 
I modified the docs.

**Comment by zcbenz** [comment]
Thanks!

**Pull request merged**